src/modules/hundred.js

```diff
@@ -0,0 +1,52 @@
+class Hundred {
+  constructor(url) {
+    this.url = url;
+  }
+
+  static createAppID = async (url) => {
+    await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        localStorage.setItem('appID', JSON.stringify(data));
+      });
+  };
+
+  // Method to fetch content of the homepage
+  getData = async (url) => {
+    await fetch(url, {
+      method: 'GET',
+    }).then((response) => response.json())
+      .then((data) => {
+        localStorage.setItem('data', JSON.stringify(data));
+      });
+  }
+
+  // Methos to fetch likes
+  getLikes = async (url) => {
```

Comment:
- [ ]  OPTIONAL
Great job so far implementing the _like_ feature! However, I think we can improve upon the UX of our _likes_ by introducing the below features to our app:
    - Like icon should toggle to indicate _like_ or _unlike_ when a particular user clicks on the like icon. If the page is refreshed, then assume that new user is present.
    - Total count of likes per user/site visitor should be 1. So, let's assume a user clicks a like, and the number of likes for a particular item should be `+1`, and if the same user clicks the like again the number of likes for that same object should be `-1`. If the page is refreshed, then assume that new user is present. 😉